demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -248,6 +260,18 @@ export class DemoMeetingApp implements
   constructor() {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     (global as any).app = this;
+
+    fatal = this.fatal.bind(this);
+
+    // Listen for unhandled errors, too.
+    window.addEventListener('error', (event) => { fatal(event.error); });
+    window.onunhandledrejection = (event: PromiseRejectionEvent) => { fatal(event.reason); };
+
+    if (document.location.search.includes('testfatal=1')) {
```

Comment:
Does that mean it will always trigger if testfatal=1?

Reply:
Yes. If you load with `testfatal=1` in the URL, a fatal is triggered automatically. This allows us to at least test that the fatal UI loads when an unhandled !